packages/interaction/src/TreeSearch.ts

```diff
@@ -1,4 +1,10 @@
 import { Point } from '@pixi/math';
+import { Container } from '@pixi/display';
+import { TilingSprite } from '@pixi/sprite-tiling';
+import { Sprite } from '@pixi/sprite';
+import { Graphics } from '@pixi/graphics';
```

Comment:
You are including new dependencies for interaction (sprite-tiling, sprite and graphics). Interaction shouldn't know about any of these display objects except Container or DisplayObject. Adding additional dependencies makes the package system a lot less flexible and interaction a lot less extendible. Please find another way to check for required APIs, this might require adding new TS class interfaces, which is okay.

Reply:
Is this a good way? cb6af43a9be6ac3eb0074978e5a2fc6c166ab4ad

Reply:
I think I've fixed it here c2a8a421dbf3491b2462609edb5d54a8b6a795fe